canonical-web-and-design/snapcraft.io
# Remove tabs in release table

## Done
Removed the tabs from the revisions table so launchpad is combined

## QA
- Go to https://snapcraft-io-3841.demos.haus/danieltest-snap/releases
- Check that there are not tabs on the "Revisions available to release" table

## Issue
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2322
